realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -232,7 +234,9 @@ public static SharedRealm getInstance(RealmConfiguration config, SchemaVersionLi
                 syncRealmUrl,
                 syncRealmAuthUrl,
                 syncUserIdentifier,
-                syncRefreshToken);
+                syncRefreshToken,
+                syncClientValidateSsl,
+                syncSslTrustCertificatePath);
```

Comment:
14 params for creating config :P 🎉 we may want to find a way to make this look better, but it is not the scope of this PR.